newIDE/app/src/MainFrame/EditorTabs/EditorTabsHandler.js

```diff
@@ -289,3 +289,24 @@ export const getEventsFunctionsExtensionEditor = (
 
   return null;
 };
+
+export const moveTabToPosition = (
+  editorTabsState: EditorTabsState,
+  fromIndex: number,
+  toIndex: number
+): EditorTabsState => {
+  if (toIndex < fromIndex) toIndex += 1;
```

Comment:
I'd always avoid modifying parameters passed to a function, and instead create new ones.
Here, you could create a new index called `destinationIndex = toIndex < fromIndex ? toIndex + 1 : toIndex`

Reply:
And I would add a comment explaining why this +1 happens. 
Something like `// If the tab is being dragged backwards, then ...`